librz/asm/p/asm_bf.c

```diff
@@ -5,15 +5,6 @@
 #include <rz_asm.h>
 
 static int disassemble(RzAsm *a, RzAsmOp *op, const ut8 *buf, int len) {
-	const ut8 *b;
-	int rep = 1;
-
-	/* Count repetitions of the current instruction, unless it's a trap. */
-	if (*buf != 0x00 && *buf != 0xff) {
-		for (b = &buf[1]; b < buf + len && *b == *buf; b++) {
-			rep++;
-		}
-	}
```

Comment:
@XVilka what about these changes?

Reply:
I guess they are unnecessary anymore?

Reply:
As you can see below they affect the ouput of `disassemble()`.

Reply:
So these changes should be moved to a separate PR.

Reply:
@pelijah I don't think it's strictly necessary. While other feedback will be addressed, this one is low priority and shouldn't stop from this PR to be merged due to the time constraints.